src/IO/H5/Python/CombineH5Dat.py

```diff
@@ -49,13 +51,19 @@ def combine_h5_dat_segments(output, input_files):
                             start_size + append_size, axis=0
                         )
                         out[dat_file_key][start_size:] = input[dat_file_key]
+                    else:
+                        logging.warning(
+                            "CombineH5DatSegments: Dat file '"
+                            + dat_file_key
```

Comment:
`f` string?

Reply:
Cool! Thanks for telling me about `f` strings. I didn't realize Python had this; I'd previously only seen something like f strings in Swift :)